tests/rust-apps-wasm/rust-sdk/cross-ctx-account/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "cross-ctx-account"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+# Build this crate as a self-contained, C-style dynamic library
+# This is required to emit the proper Wasm module type
+crate-type = ["cdylib"]
+
+[dependencies]
+# Miden SDK consists of a stdlib (intrinsic functions for VM ops, stdlib functions and types)
+# and transaction kernel API for the Miden rollup
+miden = { path = "../../../../sdk/sdk" }
+wit-bindgen-rt = "0.28"
+
+
+[profile.release]
+# optimize the output for size
+opt-level = "z"
+# Explicitly disable panic infrastructure on Wasm, as
+# there is no proper support for them anyway, and it
+# ensures that panics do not pull in a bunch of standard
+# library code unintentionally
+panic = "abort"
+
+[profile.dev]
+# Explicitly disable panic infrastructure on Wasm, as
+# there is no proper support for them anyway, and it
+# ensures that panics do not pull in a bunch of standard
+# library code unintentionally
+panic = "abort"
+opt-level = 1
+debug-assertions = false
```

Comment:
Do we really want to disable debug assertions in debug builds? Those would generally be pretty useful in tests/troubleshooting. It will result in a bit more code, but the tradeoff is that we know pretty quick if something is obviously broken.

Reply:
Enabled. I probably got carried away trying to avoid any string formatting infra for panics, etc.